src/backend/backend_ga4.py

```diff
@@ -11,12 +11,14 @@
 from .backend import TelemetryBackend
 from ..utils.cid import get_or_generate_cid, remove_cid_file
 from ..utils.params import telemetry_params
+import multiprocessing
 
 
 class GA4Backend(TelemetryBackend):
     id = 'ga4'
     cid_filename = 'openvino_ga_cid'
     old_cid_filename = 'openvino_ga_uid'
+    timeout = 1.0
```

Comment:
I think it makes sense to increase upto 3 sec.

Reply:
Done.